_layouts/blank.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Rather than duplicating content (which requires us making changes twice), can blank inherit from default or vice versa?

Reply:
One way of getting a similar result (i.e., a layout for showing of the pages themes, a layout for posts, and a blank layout for whatever other purposes) with fewer changes would be to make the default layout "plain" so as to allow the posts layout to inherit from that. Unfortunately, this would complicate usage as the default layout would no longer be the "landing page" layout anymore and to get the same "landing page" layout there would have to be a  new layout including the github project links and download links, etc. 

I was trying to follow the portion of the contribution guidelines that suggested "[t]he theme should meet the vast majority of users' needs out of the box, erring on the side of simplicity rather than flexibility, and provide users the opportunity to opt-in to additional complexity if they have specific needs or wish to further customize their experience (such as adding custom CSS or modifying the default layout)". If you're ok with me making those changes, I am more than happy to try it out!